React/NextJs/ProjectStructure.md

```diff
@@ -0,0 +1,224 @@
+# Project file structure
+
+## Organizing components
+### UI Components
+
+Before implementing [atomic design methodology](https://github.com/danilowoz/react-atomic-design), our common practice was separating components by the scope of the page + core components + shared components like this:
+
+Lowercase folders were indicators of a page scope (except of core).
+
+```
+.
+├── src
+└── components
+    ├── core
+    │   └── button
+    │       ├── Button.tsx
+    │       └── stories.tsx
+    ├── admin
+    │   └── welcome-banner
+    │       └── WelcomeBanner.tsx
+    ├── carousel
+    │   └── Carousel.tsx
+    └── card
+        └── Card.tsx
+```
+
+With atomic design, we would completely separate all UI components and build them as a blocks of atoms, molecules and organisms (together with templates and pages for putting everything together at the end) 
+
+```
+src
+.
+└── components
+    └── ui
+        ├── atoms
+        │   └── button
+        │       ├── Button.tsx
+        │       └── stories.tsx
```

Comment:
We should name this `Button.stories.tsx` because it is easier to search through the project